types/index.d.ts

```diff
@@ -4995,6 +5007,25 @@ export interface MathJsChain<TValue> {
     this: MathJsChain<T>,
     n?: number | BigNumber | MathCollection
   ): MathJsChain<T>
+  ceil<U extends MathCollection>(
+    this: MathJsChain<MathNumericType | U>,
+    n: U
+  ): MathJsChain<U>
+  ceil(this: MathJsChain<Unit>, unit: Unit): MathJsChain<Unit>
+  ceil<U extends MathCollection>(
+    this: MathJsChain<U>,
+    unit: Unit
+  ): MathJsChain<U>
+  ceil(
+    this: MathJsChain<Unit>,
+    n: number | BigNumber,
+    unit: Unit
+  ): MathJsChain<Unit>
```

Comment:
I think `ceil(Unit, number | BigNumber, Unit)` returns a `number | BigNumber` and not a `Unit`, right?

Reply:
I use the same type definition as round, which I did not update.
```
  round(
    this: MathJsChain<Unit>,
    n: number | BigNumber,
    unit: Unit
  ): MathJsChain<Unit>
  ```

Reply:
Ah, sorry my bad. I was mistaken. The functions indeed return a `Unit`.